### DISCUSSION TITLE
这是报错吗

### DISCUSSION THREAD

**Discussion by meccimiaowu** [mikumifa/biliTickerBuy#259] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/154745776/6be13c32-8707-4657-bdf7-19491004a533)

**Comment by SouthSeafireworks** [comment]
我也这样.....

**Reply by PaimonAllen** [reply]
一样429

**Reply by Nokoyoku** [reply]
v2.8.4，同429
不知道最新的v2.8.5好不好
